egs/swc/s5b/local/swc_ihm_data_prep.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash 
+
+# Copyright 2016, The University of Sheffield (Author: Yulan Liu)
+# SWC IHM data preparation (train, dev and eval), as well as scoring pareparation.
+# Apache 2.0
+
+# To be executed from one directory above this script.
+
+. path.sh
+
+export LC_ALL=C
+
+#check existing directories
+if [ $# != 2 ]; then
+  echo "Usage: swc_ihm_data_prep.sh  <path to SWC data folder>  <mode>"
+  exit 1;
+fi
+
+SWC_DIR=$1      # SWC1, SWC2, SWC3 should be in subfolders under this directory
+MODE=$2
+
+dir=data/ihm/$MODE/     # Output dir
+mkdir -p $dir
+
+
+# ---------------------------------------------------------------------------
+# Explanation of the "MODE":
+# 
+#       SA: stand-alone
+#       AD: adaptation
+#
+# SWC1, SWC2 and SWC3 all have three strips: A, B, C. 
```

Comment:
@vesis84 The A, B, C are the so-called three "strips" of each file, they are the building units of dataset. With different research focus, different dataset could be made by selecting different strips combination in train, dev and eval. I think I had explained it in the comments somewhere. Therefore the STM file has the definition of strip, and the data preparation script hard coded a few recommended dataset definition. To avoid unexpected overwriting, I extended all the folder structure by one level vertically to distinguish different datasets.